### PR TITLE
fix: columns and labels in sales-register-by-item report

### PR DESCRIPTION
**Pull request by harshmule-epi**
- Asana Task Link:

https://app.asana.com/0/1200471303892628/1200399730635326/f

* Updated Resolutions:
  * Update the Stock Qty Column Label to Quantity Sold.
  * Update the Stock UOM Column Label to Quantity UOM.
  * Removed the Excise Tax Rate, Tax Amount Columns.

### CODE REVIEW

**Review by rvpasari — DISMISSED** [review]

File: erpnext/accounts/report/sales_register_by_item/sales_register_by_item.py [hunk 1]
```diff
@@ -19,8 +19,6 @@ def _execute(filters=None, additional_table_columns=None, additional_query_colum
 	company_currency = frappe.get_cached_value('Company',  filters.get("company"),  "default_currency")
 
 	item_list = get_items(filters, additional_query_columns)
-	if item_list:
-		itemised_tax, tax_columns = get_tax_accounts(item_list, columns, company_currency)
```
Comment (vishdha) [inline erpnext/accounts/report/sales_register_by_item/sales_register_by_item.py:23]: @harshmule-git  you removed this calling function from here but at line number 310 called function still exists

File: erpnext/accounts/report/sales_register_by_item/sales_register_by_item.py [hunk 2]
```diff
@@ -406,22 +390,6 @@ def get_tax_accounts(item_list, columns, company_currency,
...
-		})
-
-		columns.append({
-			'label': _(desc + ' Amount'),
-			'fieldname': frappe.scrub(desc + ' Amount'),
-			'fieldtype': 'Currency',
-			'options': 'currency',
-			'width': 100
-		})
-
```
Comment (vishdha) [inline erpnext/accounts/report/sales_register_by_item/sales_register_by_item.py]: @harshmule-git why did you remove this chunk of code and not the entire function. because this called function does not have calling function

**Review by vishdha — CHANGES_REQUESTED** [review]

File: erpnext/accounts/report/sales_register_by_item/sales_register_by_item.py (hunk 2 above)
Comment (harshmule-epi) [reply erpnext/accounts/report/sales_register_by_item/sales_register_by_item.py]: will remove the entire function!

File: erpnext/accounts/report/sales_register_by_item/sales_register_by_item.py [hunk 3]
```diff
@@ -78,27 +76,13 @@ def _execute(filters=None, additional_table_columns=None, additional_query_colum
...
-		})
 
 		if filters.get('group_by'):
 			row.update({'percent_gt': flt(row['total']/grand_total) * 100})
 			group_by_field, subtotal_display_field = get_group_by_and_display_fields(filters)
 			data, prev_group_by_value = add_total_row(data, filters, prev_group_by_value, d, total_row_map,
-				group_by_field, subtotal_display_field, grand_total, tax_columns)
-			add_sub_total_row(row, total_row_map, d.get(group_by_field, ''), tax_columns)
+				group_by_field, subtotal_display_field, grand_total)
+			add_sub_total_row(row, total_row_map, d.get(group_by_field, ''))
```
Comment (vishdha) [inline erpnext/accounts/report/sales_register_by_item/sales_register_by_item.py:85]: @harshmule-git did you tested locally. please test and review from start to end each and every calling function.
add_total_row contain tax_column when its defined should be removed from there too

Comment (vishdha) [inline erpnext/accounts/report/sales_register_by_item/sales_register_by_item.py:85]: add_sub_total_row also contain args tax_column at its called function.

**Review by vishdha — CHANGES_REQUESTED** [review]

File: erpnext/accounts/report/sales_register_by_item/sales_register_by_item.py (hunk 3 above)
Comment (harshmule-epi) [reply erpnext/accounts/report/sales_register_by_item/sales_register_by_item.py:85]: it's returning none, so it was working at my end, if needed i can remove the args!

**Comment by hrwX** [comment]
@harshmule-git @vishdha  Whats the status of this PR ?

**Comment by harshmule-epi** [comment]
Hey @hrwX already pinged and asked raj for review
waiting for that!!
Ps: All the changes are done!

**Review by vishdha — APPROVED** [review]

**Comment by sahil28297** [comment]
@harshmule-git Rebase this to bs-jul-sprint-1, if the code is not present there, then rebase it to pre-release

File: erpnext/accounts/report/sales_register_by_item/sales_register_by_item.py [hunk 4]
```diff
@@ -323,116 +307,14 @@ def get_grand_total(filters, doctype):
 def get_deducted_taxes():
 	return frappe.db.sql_list("select name from `tabPurchase Taxes and Charges` where add_deduct_tax = 'Deduct'")
 
-def get_tax_accounts(item_list, columns, company_currency,
```
Comment (sahil28297) [inline erpnext/accounts/report/sales_register_by_item/sales_register_by_item.py]: This function is imported in `item_wise_purchase_register.py`

**Review by sahil28297 — CHANGES_REQUESTED** [review]

File: erpnext/accounts/report/sales_register_by_item/sales_register_by_item.py (hunk 4 above)
Comment (harshmule-epi) [reply erpnext/accounts/report/sales_register_by_item/sales_register_by_item.py]: will revert it!

**Comment by harshmule-epi** [comment]
hey @sahil28297 new PR #1437 is created for this issue with pre-release as base, shall we close this PR?
